modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,62 @@
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
+[#release-701]
+== Release 7.0.1 (September 2021)
+
+Couchbase Server 7.0.1 was released in September 2021.
+This maintenance release contains bug fixes.
+
+[#fixed-issues-701]
+=== Fixed Issues
+
+This section highlights the notable issues fixed in this release.
+
+==== Operator
+
+[#table_fixedissues_v701-operator,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-47678[MB-47678^]
+| *Summary*: Fixed an error occurring during the refresh of the FTS indexer.
```

Comment:
This should be reworded to say something like "Fixed an error occurring when running Flex index queries on a setup that uses alternate addresses"